kernel/Dockerfile

```diff
@@ -1,4 +1,6 @@
-FROM linuxkit/alpine:e2391e0b164c57db9f6c4ae110ee84f766edc430 AS kernel-build
+ARG BUILDER
```

Comment:
`BUILDER` is an odd name. Maybe `BUILD_BASE` or `BUILD_IMAGE` is more descriptive.

Reply:
I like `BUILD_IMAGE`. Done.

Reply:
Updated